spec/facades/listing_facade_spec.rb

```diff
@@ -38,6 +39,22 @@
           expect(ListingFacade.data_hash([])).to eq(nil)
         end
       end
+
```

Comment:
This section is testing a stubbed version of the API calls. should it be replaced with a real API call and a vcr?